html/semantics/forms/event-propagate-disabled.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<title>Event propagation on disabled form elements</title>
+<meta charset=utf8>
+<meta name=timeout content=long>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<body data-expected="mousedown,mouseup">
+<div id=cases>
+<input data-expected="mousedown,mouseup,click"> <!-- Santity check with non-disabled control -->
+<select disabled="disabled" data-expected=""></select>
+<select disabled="disabled" data-expected=""><option selected="selected">foo</option></select>
+<fieldset disabled="" data-expected="">Text</fieldset>
+<fieldset disabled="" data-expected=""><span>Span</span></fieldset>
```

Comment:
Does the `<span>` get clicked in this test? Based on the code it seems to me like only direct children of `<div id=cases>` are getting clicked